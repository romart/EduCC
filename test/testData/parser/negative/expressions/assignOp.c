int i;
float f;
int *pi;
float *pf;

int arr[20];

struct S { int good; };

struct S s, *ps;

void testAsg() {
  i = i;
  i = f;
  i = pi;
  i = pf;
  i = arr;
  i = s;
  i = ps;
  f = i;
  f = f;
  f = pi;
  f = pf;
  f = arr;
  f = s;
  f = ps;
  pi = i;
  pi = f;
  pi = pi;
  pi = pf;
  pi = arr;
  pi = s;
  pi = ps;
  pf = i;
  pf = f;
  pf = pi;
  pf = pf;
  pf = arr;
  pf = s;
  pf = ps;
  arr = i;
  arr = f;
  arr = pi;
  arr = pf;
  arr = arr;
  arr = s;
  arr = ps;
  s = i;
  s = f;
  s = pi;
  s = pf;
  s = arr;
  s = s;
  s = ps;
  ps = i;
  ps = f;
  ps = pi;
  ps = pf;
  ps = arr;
  ps = s;
  ps = ps;
}


void testPlusAsg() {
  i += i;
  i += f;
  i += pi;
  i += pf;
  i += arr;
  i += s;
  i += ps;
  f += i;
  f += f;
  f += pi;
  f += pf;
  f += arr;
  f += s;
  f += ps;
  pi += i;
  pi += f;
  pi += pi;
  pi += pf;
  pi += arr;
  pi += s;
  pi += ps;
  pf += i;
  pf += f;
  pf += pi;
  pf += pf;
  pf += arr;
  pf += s;
  pf += ps;
  arr += i;
  arr += f;
  arr += pi;
  arr += pf;
  arr += arr;
  arr += s;
  arr += ps;
  s += i;
  s += f;
  s += pi;
  s += pf;
  s += arr;
  s += s;
  s += ps;
  ps += i;
  ps += f;
  ps += pi;
  ps += pf;
  ps += arr;
  ps += s;
  ps += ps;
}


void testMinusAsg() {
  i -= i;
  i -= f;
  i -= pi;
  i -= pf;
  i -= arr;
  i -= s;
  i -= ps;
  f -= i;
  f -= f;
  f -= pi;
  f -= pf;
  f -= arr;
  f -= s;
  f -= ps;
  pi -= i;
  pi -= f;
  pi -= pi;
  pi -= pf;
  pi -= arr;
  pi -= s;
  pi -= ps;
  pf -= i;
  pf -= f;
  pf -= pi;
  pf -= pf;
  pf -= arr;
  pf -= s;
  pf -= ps;
  arr -= i;
  arr -= f;
  arr -= pi;
  arr -= pf;
  arr -= arr;
  arr -= s;
  arr -= ps;
  s -= i;
  s -= f;
  s -= pi;
  s -= pf;
  s -= arr;
  s -= s;
  s -= ps;
  ps -= i;
  ps -= f;
  ps -= pi;
  ps -= pf;
  ps -= arr;
  ps -= s;
  ps -= ps;
}


void testMulAsg() {
  i *= i;
  i *= f;
  i *= pi;
  i *= pf;
  i *= arr;
  i *= s;
  i *= ps;
  f *= i;
  f *= f;
  f *= pi;
  f *= pf;
  f *= arr;
  f *= s;
  f *= ps;
  pi *= i;
  pi *= f;
  pi *= pi;
  pi *= pf;
  pi *= arr;
  pi *= s;
  pi *= ps;
  pf *= i;
  pf *= f;
  pf *= pi;
  pf *= pf;
  pf *= arr;
  pf *= s;
  pf *= ps;
  arr *= i;
  arr *= f;
  arr *= pi;
  arr *= pf;
  arr *= arr;
  arr *= s;
  arr *= ps;
  s *= i;
  s *= f;
  s *= pi;
  s *= pf;
  s *= arr;
  s *= s;
  s *= ps;
  ps *= i;
  ps *= f;
  ps *= pi;
  ps *= pf;
  ps *= arr;
  ps *= s;
  ps *= ps;
}


void testDivAsg() {
  i /= i;
  i /= f;
  i /= pi;
  i /= pf;
  i /= arr;
  i /= s;
  i /= ps;
  f /= i;
  f /= f;
  f /= pi;
  f /= pf;
  f /= arr;
  f /= s;
  f /= ps;
  pi /= i;
  pi /= f;
  pi /= pi;
  pi /= pf;
  pi /= arr;
  pi /= s;
  pi /= ps;
  pf /= i;
  pf /= f;
  pf /= pi;
  pf /= pf;
  pf /= arr;
  pf /= s;
  pf /= ps;
  arr /= i;
  arr /= f;
  arr /= pi;
  arr /= pf;
  arr /= arr;
  arr /= s;
  arr /= ps;
  s /= i;
  s /= f;
  s /= pi;
  s /= pf;
  s /= arr;
  s /= s;
  s /= ps;
  ps /= i;
  ps /= f;
  ps /= pi;
  ps /= pf;
  ps /= arr;
  ps /= s;
  ps /= ps;
}


void testModAsg() {
  i %= i;
  i %= f;
  i %= pi;
  i %= pf;
  i %= arr;
  i %= s;
  i %= ps;
  f %= i;
  f %= f;
  f %= pi;
  f %= pf;
  f %= arr;
  f %= s;
  f %= ps;
  pi %= i;
  pi %= f;
  pi %= pi;
  pi %= pf;
  pi %= arr;
  pi %= s;
  pi %= ps;
  pf %= i;
  pf %= f;
  pf %= pi;
  pf %= pf;
  pf %= arr;
  pf %= s;
  pf %= ps;
  arr %= i;
  arr %= f;
  arr %= pi;
  arr %= pf;
  arr %= arr;
  arr %= s;
  arr %= ps;
  s %= i;
  s %= f;
  s %= pi;
  s %= pf;
  s %= arr;
  s %= s;
  s %= ps;
  ps %= i;
  ps %= f;
  ps %= pi;
  ps %= pf;
  ps %= arr;
  ps %= s;
  ps %= ps;
}


void testLhsAsg() {
  i <<= i;
  i <<= f;
  i <<= pi;
  i <<= pf;
  i <<= arr;
  i <<= s;
  i <<= ps;
  f <<= i;
  f <<= f;
  f <<= pi;
  f <<= pf;
  f <<= arr;
  f <<= s;
  f <<= ps;
  pi <<= i;
  pi <<= f;
  pi <<= pi;
  pi <<= pf;
  pi <<= arr;
  pi <<= s;
  pi <<= ps;
  pf <<= i;
  pf <<= f;
  pf <<= pi;
  pf <<= pf;
  pf <<= arr;
  pf <<= s;
  pf <<= ps;
  arr <<= i;
  arr <<= f;
  arr <<= pi;
  arr <<= pf;
  arr <<= arr;
  arr <<= s;
  arr <<= ps;
  s <<= i;
  s <<= f;
  s <<= pi;
  s <<= pf;
  s <<= arr;
  s <<= s;
  s <<= ps;
  ps <<= i;
  ps <<= f;
  ps <<= pi;
  ps <<= pf;
  ps <<= arr;
  ps <<= s;
  ps <<= ps;
}


void testRhsAsg() {
  i >>= i;
  i >>= f;
  i >>= pi;
  i >>= pf;
  i >>= arr;
  i >>= s;
  i >>= ps;
  f >>= i;
  f >>= f;
  f >>= pi;
  f >>= pf;
  f >>= arr;
  f >>= s;
  f >>= ps;
  pi >>= i;
  pi >>= f;
  pi >>= pi;
  pi >>= pf;
  pi >>= arr;
  pi >>= s;
  pi >>= ps;
  pf >>= i;
  pf >>= f;
  pf >>= pi;
  pf >>= pf;
  pf >>= arr;
  pf >>= s;
  pf >>= ps;
  arr >>= i;
  arr >>= f;
  arr >>= pi;
  arr >>= pf;
  arr >>= arr;
  arr >>= s;
  arr >>= ps;
  s >>= i;
  s >>= f;
  s >>= pi;
  s >>= pf;
  s >>= arr;
  s >>= s;
  s >>= ps;
  ps >>= i;
  ps >>= f;
  ps >>= pi;
  ps >>= pf;
  ps >>= arr;
  ps >>= s;
  ps >>= ps;
}


void testAndAsg() {
  i &= i;
  i &= f;
  i &= pi;
  i &= pf;
  i &= arr;
  i &= s;
  i &= ps;
  f &= i;
  f &= f;
  f &= pi;
  f &= pf;
  f &= arr;
  f &= s;
  f &= ps;
  pi &= i;
  pi &= f;
  pi &= pi;
  pi &= pf;
  pi &= arr;
  pi &= s;
  pi &= ps;
  pf &= i;
  pf &= f;
  pf &= pi;
  pf &= pf;
  pf &= arr;
  pf &= s;
  pf &= ps;
  arr &= i;
  arr &= f;
  arr &= pi;
  arr &= pf;
  arr &= arr;
  arr &= s;
  arr &= ps;
  s &= i;
  s &= f;
  s &= pi;
  s &= pf;
  s &= arr;
  s &= s;
  s &= ps;
  ps &= i;
  ps &= f;
  ps &= pi;
  ps &= pf;
  ps &= arr;
  ps &= s;
  ps &= ps;
}


void testOrAsg() {
  i |= i;
  i |= f;
  i |= pi;
  i |= pf;
  i |= arr;
  i |= s;
  i |= ps;
  f |= i;
  f |= f;
  f |= pi;
  f |= pf;
  f |= arr;
  f |= s;
  f |= ps;
  pi |= i;
  pi |= f;
  pi |= pi;
  pi |= pf;
  pi |= arr;
  pi |= s;
  pi |= ps;
  pf |= i;
  pf |= f;
  pf |= pi;
  pf |= pf;
  pf |= arr;
  pf |= s;
  pf |= ps;
  arr |= i;
  arr |= f;
  arr |= pi;
  arr |= pf;
  arr |= arr;
  arr |= s;
  arr |= ps;
  s |= i;
  s |= f;
  s |= pi;
  s |= pf;
  s |= arr;
  s |= s;
  s |= ps;
  ps |= i;
  ps |= f;
  ps |= pi;
  ps |= pf;
  ps |= arr;
  ps |= s;
  ps |= ps;
}


void testXorAsg() {
  i ^= i;
  i ^= f;
  i ^= pi;
  i ^= pf;
  i ^= arr;
  i ^= s;
  i ^= ps;
  f ^= i;
  f ^= f;
  f ^= pi;
  f ^= pf;
  f ^= arr;
  f ^= s;
  f ^= ps;
  pi ^= i;
  pi ^= f;
  pi ^= pi;
  pi ^= pf;
  pi ^= arr;
  pi ^= s;
  pi ^= ps;
  pf ^= i;
  pf ^= f;
  pf ^= pi;
  pf ^= pf;
  pf ^= arr;
  pf ^= s;
  pf ^= ps;
  arr ^= i;
  arr ^= f;
  arr ^= pi;
  arr ^= pf;
  arr ^= arr;
  arr ^= s;
  arr ^= ps;
  s ^= i;
  s ^= f;
  s ^= pi;
  s ^= pf;
  s ^= arr;
  s ^= s;
  s ^= ps;
  ps ^= i;
  ps ^= f;
  ps ^= pi;
  ps ^= pf;
  ps ^= arr;
  ps ^= s;
  ps ^= ps;
}
