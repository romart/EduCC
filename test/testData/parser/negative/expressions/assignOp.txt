FILE test/testData/parser/negative/expressions/assignOp.c
  signed int i
----
  float f
----
  *signed int pi
----
  *float pf
----
  signed int[20] arr
----
  STRUCT S
    signed int good
  STRUCT_END
----
  STRUCT S s
----
  *STRUCT S ps
----
  FUN void testAsg 
  BEGIN
    i = i
    i = f
    i = pi
    i = pf
    i = arr
    i = s
    i = ps
    f = i
    f = f
    f = pi
    f = pf
    f = arr
    f = s
    f = ps
    pi = i
    pi = f
    pi = pi
    pi = pf
    pi = arr
    pi = s
    pi = ps
    pf = i
    pf = f
    pf = pi
    pf = pf
    pf = arr
    pf = s
    pf = ps
    arr = i
    arr = f
    arr = pi
    arr = pf
    arr = arr
    arr = s
    arr = ps
    s = i
    s = f
    s = pi
    s = pf
    s = arr
    s = s
    s = ps
    ps = i
    ps = f
    ps = pi
    ps = pf
    ps = arr
    ps = s
    ps = ps    
  END
----
  FUN void testPlusAsg 
  BEGIN
    i = i + i
    i = i + f
    i = i + pi
    i = i + pf
    i = i + arr
    i = i + s
    i = i + ps
    f = f + i
    f = f + f
    f = f + pi
    f = f + pf
    f = f + arr
    f = f + s
    f = f + ps
    pi = pi + i
    pi = pi + f
    pi = pi + pi
    pi = pi + pf
    pi = pi + arr
    pi = pi + s
    pi = pi + ps
    pf = pf + i
    pf = pf + f
    pf = pf + pi
    pf = pf + pf
    pf = pf + arr
    pf = pf + s
    pf = pf + ps
    arr = arr + i
    arr = arr + f
    arr = arr + pi
    arr = arr + pf
    arr = arr + arr
    arr = arr + s
    arr = arr + ps
    s = s + i
    s = s + f
    s = s + pi
    s = s + pf
    s = s + arr
    s = s + s
    s = s + ps
    ps = ps + i
    ps = ps + f
    ps = ps + pi
    ps = ps + pf
    ps = ps + arr
    ps = ps + s
    ps = ps + ps    
  END
----
  FUN void testMinusAsg 
  BEGIN
    i = i - i
    i = i - f
    i = i - pi
    i = i - pf
    i = i - arr
    i = i - s
    i = i - ps
    f = f - i
    f = f - f
    f = f - pi
    f = f - pf
    f = f - arr
    f = f - s
    f = f - ps
    pi = pi - i
    pi = pi - f
    pi = pi - pi
    pi = pi - pf
    pi = pi - arr
    pi = pi - s
    pi = pi - ps
    pf = pf - i
    pf = pf - f
    pf = pf - pi
    pf = pf - pf
    pf = pf - arr
    pf = pf - s
    pf = pf - ps
    arr = arr - i
    arr = arr - f
    arr = arr - pi
    arr = arr - pf
    arr = arr - arr
    arr = arr - s
    arr = arr - ps
    s = s - i
    s = s - f
    s = s - pi
    s = s - pf
    s = s - arr
    s = s - s
    s = s - ps
    ps = ps - i
    ps = ps - f
    ps = ps - pi
    ps = ps - pf
    ps = ps - arr
    ps = ps - s
    ps = ps - ps    
  END
----
  FUN void testMulAsg 
  BEGIN
    i = i * i
    i = i * f
    i = i * pi
    i = i * pf
    i = i * arr
    i = i * s
    i = i * ps
    f = f * i
    f = f * f
    f = f * pi
    f = f * pf
    f = f * arr
    f = f * s
    f = f * ps
    pi = pi * i
    pi = pi * f
    pi = pi * pi
    pi = pi * pf
    pi = pi * arr
    pi = pi * s
    pi = pi * ps
    pf = pf * i
    pf = pf * f
    pf = pf * pi
    pf = pf * pf
    pf = pf * arr
    pf = pf * s
    pf = pf * ps
    arr = arr * i
    arr = arr * f
    arr = arr * pi
    arr = arr * pf
    arr = arr * arr
    arr = arr * s
    arr = arr * ps
    s = s * i
    s = s * f
    s = s * pi
    s = s * pf
    s = s * arr
    s = s * s
    s = s * ps
    ps = ps * i
    ps = ps * f
    ps = ps * pi
    ps = ps * pf
    ps = ps * arr
    ps = ps * s
    ps = ps * ps    
  END
----
  FUN void testDivAsg 
  BEGIN
    i = i / i
    i = i / f
    i = i / pi
    i = i / pf
    i = i / arr
    i = i / s
    i = i / ps
    f = f / i
    f = f / f
    f = f / pi
    f = f / pf
    f = f / arr
    f = f / s
    f = f / ps
    pi = pi / i
    pi = pi / f
    pi = pi / pi
    pi = pi / pf
    pi = pi / arr
    pi = pi / s
    pi = pi / ps
    pf = pf / i
    pf = pf / f
    pf = pf / pi
    pf = pf / pf
    pf = pf / arr
    pf = pf / s
    pf = pf / ps
    arr = arr / i
    arr = arr / f
    arr = arr / pi
    arr = arr / pf
    arr = arr / arr
    arr = arr / s
    arr = arr / ps
    s = s / i
    s = s / f
    s = s / pi
    s = s / pf
    s = s / arr
    s = s / s
    s = s / ps
    ps = ps / i
    ps = ps / f
    ps = ps / pi
    ps = ps / pf
    ps = ps / arr
    ps = ps / s
    ps = ps / ps    
  END
----
  FUN void testModAsg 
  BEGIN
    i = i % i
    i = i % f
    i = i % pi
    i = i % pf
    i = i % arr
    i = i % s
    i = i % ps
    f = f % i
    f = f % f
    f = f % pi
    f = f % pf
    f = f % arr
    f = f % s
    f = f % ps
    pi = pi % i
    pi = pi % f
    pi = pi % pi
    pi = pi % pf
    pi = pi % arr
    pi = pi % s
    pi = pi % ps
    pf = pf % i
    pf = pf % f
    pf = pf % pi
    pf = pf % pf
    pf = pf % arr
    pf = pf % s
    pf = pf % ps
    arr = arr % i
    arr = arr % f
    arr = arr % pi
    arr = arr % pf
    arr = arr % arr
    arr = arr % s
    arr = arr % ps
    s = s % i
    s = s % f
    s = s % pi
    s = s % pf
    s = s % arr
    s = s % s
    s = s % ps
    ps = ps % i
    ps = ps % f
    ps = ps % pi
    ps = ps % pf
    ps = ps % arr
    ps = ps % s
    ps = ps % ps    
  END
----
  FUN void testLhsAsg 
  BEGIN
    i = i << i
    i = i << f
    i = i << pi
    i = i << pf
    i = i << arr
    i = i << s
    i = i << ps
    f = f << i
    f = f << f
    f = f << pi
    f = f << pf
    f = f << arr
    f = f << s
    f = f << ps
    pi = pi << i
    pi = pi << f
    pi = pi << pi
    pi = pi << pf
    pi = pi << arr
    pi = pi << s
    pi = pi << ps
    pf = pf << i
    pf = pf << f
    pf = pf << pi
    pf = pf << pf
    pf = pf << arr
    pf = pf << s
    pf = pf << ps
    arr = arr << i
    arr = arr << f
    arr = arr << pi
    arr = arr << pf
    arr = arr << arr
    arr = arr << s
    arr = arr << ps
    s = s << i
    s = s << f
    s = s << pi
    s = s << pf
    s = s << arr
    s = s << s
    s = s << ps
    ps = ps << i
    ps = ps << f
    ps = ps << pi
    ps = ps << pf
    ps = ps << arr
    ps = ps << s
    ps = ps << ps    
  END
----
  FUN void testRhsAsg 
  BEGIN
    i = i >> i
    i = i >> f
    i = i >> pi
    i = i >> pf
    i = i >> arr
    i = i >> s
    i = i >> ps
    f = f >> i
    f = f >> f
    f = f >> pi
    f = f >> pf
    f = f >> arr
    f = f >> s
    f = f >> ps
    pi = pi >> i
    pi = pi >> f
    pi = pi >> pi
    pi = pi >> pf
    pi = pi >> arr
    pi = pi >> s
    pi = pi >> ps
    pf = pf >> i
    pf = pf >> f
    pf = pf >> pi
    pf = pf >> pf
    pf = pf >> arr
    pf = pf >> s
    pf = pf >> ps
    arr = arr >> i
    arr = arr >> f
    arr = arr >> pi
    arr = arr >> pf
    arr = arr >> arr
    arr = arr >> s
    arr = arr >> ps
    s = s >> i
    s = s >> f
    s = s >> pi
    s = s >> pf
    s = s >> arr
    s = s >> s
    s = s >> ps
    ps = ps >> i
    ps = ps >> f
    ps = ps >> pi
    ps = ps >> pf
    ps = ps >> arr
    ps = ps >> s
    ps = ps >> ps    
  END
----
  FUN void testAndAsg 
  BEGIN
    i = i & i
    i = i & f
    i = i & pi
    i = i & pf
    i = i & arr
    i = i & s
    i = i & ps
    f = f & i
    f = f & f
    f = f & pi
    f = f & pf
    f = f & arr
    f = f & s
    f = f & ps
    pi = pi & i
    pi = pi & f
    pi = pi & pi
    pi = pi & pf
    pi = pi & arr
    pi = pi & s
    pi = pi & ps
    pf = pf & i
    pf = pf & f
    pf = pf & pi
    pf = pf & pf
    pf = pf & arr
    pf = pf & s
    pf = pf & ps
    arr = arr & i
    arr = arr & f
    arr = arr & pi
    arr = arr & pf
    arr = arr & arr
    arr = arr & s
    arr = arr & ps
    s = s & i
    s = s & f
    s = s & pi
    s = s & pf
    s = s & arr
    s = s & s
    s = s & ps
    ps = ps & i
    ps = ps & f
    ps = ps & pi
    ps = ps & pf
    ps = ps & arr
    ps = ps & s
    ps = ps & ps    
  END
----
  FUN void testOrAsg 
  BEGIN
    i = i | i
    i = i | f
    i = i | pi
    i = i | pf
    i = i | arr
    i = i | s
    i = i | ps
    f = f | i
    f = f | f
    f = f | pi
    f = f | pf
    f = f | arr
    f = f | s
    f = f | ps
    pi = pi | i
    pi = pi | f
    pi = pi | pi
    pi = pi | pf
    pi = pi | arr
    pi = pi | s
    pi = pi | ps
    pf = pf | i
    pf = pf | f
    pf = pf | pi
    pf = pf | pf
    pf = pf | arr
    pf = pf | s
    pf = pf | ps
    arr = arr | i
    arr = arr | f
    arr = arr | pi
    arr = arr | pf
    arr = arr | arr
    arr = arr | s
    arr = arr | ps
    s = s | i
    s = s | f
    s = s | pi
    s = s | pf
    s = s | arr
    s = s | s
    s = s | ps
    ps = ps | i
    ps = ps | f
    ps = ps | pi
    ps = ps | pf
    ps = ps | arr
    ps = ps | s
    ps = ps | ps    
  END
----
  FUN void testXorAsg 
  BEGIN
    i = i ^ i
    i = i ^ f
    i = i ^ pi
    i = i ^ pf
    i = i ^ arr
    i = i ^ s
    i = i ^ ps
    f = f ^ i
    f = f ^ f
    f = f ^ pi
    f = f ^ pf
    f = f ^ arr
    f = f ^ s
    f = f ^ ps
    pi = pi ^ i
    pi = pi ^ f
    pi = pi ^ pi
    pi = pi ^ pf
    pi = pi ^ arr
    pi = pi ^ s
    pi = pi ^ ps
    pf = pf ^ i
    pf = pf ^ f
    pf = pf ^ pi
    pf = pf ^ pf
    pf = pf ^ arr
    pf = pf ^ s
    pf = pf ^ ps
    arr = arr ^ i
    arr = arr ^ f
    arr = arr ^ pi
    arr = arr ^ pf
    arr = arr ^ arr
    arr = arr ^ s
    arr = arr ^ ps
    s = s ^ i
    s = s ^ f
    s = s ^ pi
    s = s ^ pf
    s = s ^ arr
    s = s ^ s
    s = s ^ ps
    ps = ps ^ i
    ps = ps ^ f
    ps = ps ^ pi
    ps = ps ^ pf
    ps = ps ^ arr
    ps = ps ^ s
    ps = ps ^ ps    
  END