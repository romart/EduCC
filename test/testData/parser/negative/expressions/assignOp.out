test/testData/parser/negative/expressions/assignOp.c:5:15: warning: incompatible integer to pointer conversion assigning to 'signed int' from '*signed int'
test/testData/parser/negative/expressions/assignOp.c:5:16: warning: incompatible integer to pointer conversion assigning to 'signed int' from '*float'
test/testData/parser/negative/expressions/assignOp.c:5:17: warning: incompatible integer to pointer conversion assigning to 'signed int' from 'signed int[20]'
test/testData/parser/negative/expressions/assignOp.c:5:19: warning: incompatible integer to pointer conversion assigning to 'signed int' from '*STRUCT S'
test/testData/parser/negative/expressions/assignOp.c:6:27: warning: incompatible integer to pointer conversion assigning to '*signed int' from 'signed int'
test/testData/parser/negative/expressions/assignOp.c:6:30: warning: incompatible pointer types assigning to '*signed int' from '*float'
test/testData/parser/negative/expressions/assignOp.c:6:33: warning: incompatible pointer types assigning to '*signed int' from '*STRUCT S'
test/testData/parser/negative/expressions/assignOp.c:6:34: warning: incompatible integer to pointer conversion assigning to '*float' from 'signed int'
test/testData/parser/negative/expressions/assignOp.c:6:36: warning: incompatible pointer types assigning to '*float' from '*signed int'
test/testData/parser/negative/expressions/assignOp.c:6:38: warning: incompatible pointer types assigning to '*float' from 'signed int[20]'
test/testData/parser/negative/expressions/assignOp.c:6:40: warning: incompatible pointer types assigning to '*float' from '*STRUCT S'
test/testData/parser/negative/expressions/assignOp.c:6:55: warning: incompatible integer to pointer conversion assigning to '*STRUCT S' from 'signed int'
test/testData/parser/negative/expressions/assignOp.c:6:57: warning: incompatible pointer types assigning to '*STRUCT S' from '*signed int'
test/testData/parser/negative/expressions/assignOp.c:6:58: warning: incompatible pointer types assigning to '*STRUCT S' from '*float'
test/testData/parser/negative/expressions/assignOp.c:6:59: warning: incompatible pointer types assigning to '*STRUCT S' from 'signed int[20]'
test/testData/parser/negative/expressions/assignOp.c:5:68: warning: incompatible integer to pointer conversion assigning to 'signed int' from '*signed int'
test/testData/parser/negative/expressions/assignOp.c:5:69: warning: incompatible integer to pointer conversion assigning to 'signed int' from '*float'
test/testData/parser/negative/expressions/assignOp.c:5:70: warning: incompatible integer to pointer conversion assigning to 'signed int' from '*signed int'
test/testData/parser/negative/expressions/assignOp.c:5:72: warning: incompatible integer to pointer conversion assigning to 'signed int' from '*STRUCT S'
test/testData/parser/negative/expressions/assignOp.c:5:121: warning: incompatible integer to pointer conversion assigning to 'signed int' from '*signed int'
test/testData/parser/negative/expressions/assignOp.c:5:122: warning: incompatible integer to pointer conversion assigning to 'signed int' from '*float'
test/testData/parser/negative/expressions/assignOp.c:5:123: warning: incompatible integer to pointer conversion assigning to 'signed int' from '*signed int'
test/testData/parser/negative/expressions/assignOp.c:5:125: warning: incompatible integer to pointer conversion assigning to 'signed int' from '*STRUCT S'
test/testData/parser/negative/expressions/assignOp.c:6:135: warning: incompatible integer to pointer conversion assigning to '*signed int' from 'signed long'
test/testData/parser/negative/expressions/assignOp.c:6:136: warning: incompatible integer to pointer conversion assigning to '*signed int' from 'signed long'
test/testData/parser/negative/expressions/assignOp.c:6:137: warning: incompatible integer to pointer conversion assigning to '*signed int' from 'signed long'
test/testData/parser/negative/expressions/assignOp.c:6:139: warning: incompatible integer to pointer conversion assigning to '*signed int' from 'signed long'
test/testData/parser/negative/expressions/assignOp.c:6:142: warning: incompatible integer to pointer conversion assigning to '*float' from 'signed long'
test/testData/parser/negative/expressions/assignOp.c:6:143: warning: incompatible integer to pointer conversion assigning to '*float' from 'signed long'
test/testData/parser/negative/expressions/assignOp.c:6:144: warning: incompatible integer to pointer conversion assigning to '*float' from 'signed long'
test/testData/parser/negative/expressions/assignOp.c:6:146: warning: incompatible integer to pointer conversion assigning to '*float' from 'signed long'
test/testData/parser/negative/expressions/assignOp.c:6:163: warning: incompatible integer to pointer conversion assigning to '*STRUCT S' from 'signed long'
test/testData/parser/negative/expressions/assignOp.c:6:164: warning: incompatible integer to pointer conversion assigning to '*STRUCT S' from 'signed long'
test/testData/parser/negative/expressions/assignOp.c:6:165: warning: incompatible integer to pointer conversion assigning to '*STRUCT S' from 'signed long'
test/testData/parser/negative/expressions/assignOp.c:6:167: warning: incompatible integer to pointer conversion assigning to '*STRUCT S' from 'signed long'
